src/Chapter10/Listing10.05.OverridingEquals.cs

```diff
@@ -60,22 +59,20 @@ public bool Equals(Coordinate obj)
         {
             // STEP 1: Check for null if a reference type
             // (e.g., a reference type)
-            // if (obj == null)
+            // if (ReferenceEquals(obj, null)
```

Comment:
I realize this matches the manuscript, but IMO it would be better if this suggested using pattern matching for the null check to avoid boxing. @MarkMichaelis 
```suggestion
            // if (obj is null)
```

Reply:
Needs to be done throughout the book, but changed it here